src/ucp/core/ucp_request.h

```diff
@@ -125,7 +128,10 @@ struct ucp_request {
          * operations */
         struct {
             ucp_ep_h                ep;
-            void                    *buffer;    /* Send buffer */
+            union {
+                void                   *buffer; /* Send buffer */
+                ucp_request_callback_t flushed_cb;/* Called when flushed */
+            };
```

Comment:
how is it related to request id refactoring?

Reply:
to not increase UCP request size

Reply:
ok